### PR TITLE
Add formula for wiki

### PR DESCRIPTION
**Pull request by walle**
Got a request for packaging my tool with homebrew (https://github.com/walle/wiki/issues/10)

The tool is called wiki (https://github.com/walle/wiki) and built with Go.

I hope I got everything right in the formula. I have run `brew audit wiki`, `brew tests` and `brew install wiki && brew test wiki` without any issues.

I have also run `brew audit --strict wiki` and fixed all issues.

Please tell me if there is anything else I have to fix.

### CODE REVIEW

File: Library/Formula/wiki.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+require "language/go"
+
+class Wiki < Formula
+  desc "Fetch summaries from MediaWiki wikis, like Wikipedia"
+  homepage "https://github.com/walle/wiki"
+  url "https://github.com/walle/wikiarchive/1.3.0.tar.gz"
```
Comment (bfontaine) [inline Library/Formula/wiki.rb]: You missed a slash here: `wikiarchive` → `wiki/archive`.

File: Library/Formula/wiki.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+    cd "src/github.com/walle/wiki" do
+      system "make"
+      bin.install "build/wiki"
+      man1.install "_doc/wiki.1"
+    end
+  end
+
+  test do
+    system bin/"wiki", "-version"
```
Comment (bfontaine) [inline Library/Formula/wiki.rb]: Can’t we write a test that does something more substantial than e.g. `-help`/`-version`?

File: Library/Formula/wiki.rb (hunk 1 above)
Comment (walle) [inline Library/Formula/wiki.rb]: Thanks, I have updated the pull request.

File: Library/Formula/wiki.rb (hunk 2 above)
Comment (walle) [inline Library/Formula/wiki.rb]: Do you have an example? 

I have this test for the tool, would it be possible to convert it to a ruby test somehow?

```bash
# Test that standard usage prints a link to page
OUTPUT="$($BIN golang)"
STATUS=$?
if [[ $STATUS -ne 0 ]]; then
  fail 'Did not get success exit code'
  exit 1
fi
echo "$OUTPUT" | grep -q "Read more: https://en.wikipedia.org/wiki/Go"
if [ $? -ne 0 ];then
  fail 'Standard usage did not output link to page'
fi
pass
```

Comment (bfontaine) [inline Library/Formula/wiki.rb]: Something like this should work:

```rb
assert_match "Read more: https://en.wikipedia.org/wiki/Go", shell_output("#{bin}/wiki golang")
```

Comment (walle) [inline Library/Formula/wiki.rb]: Thanks! I have updated the test.

**Comment by DomT4** [comment]
The only remaining failure is this:
```
==> audit problems
wiki:
 * GitHub repository too new (<30 days old)
```

Given the upstream popularity of this project, even with being very new, I'm open to some flexibility on the requirement there unless other maintainers are :-1: on the idea.

**Comment by bfontaine** [comment]
> Given the upstream popularity of this project, even with being very new, I'm open to some flexibility on the requirement

I’m :+1: on that because it’s popular _and_ we have a stable release.

**Comment by walle** [comment]
I noticed one thing, and will update the pull request.

The formula uses `make [build]`, which does a `go get` for a dependency. I think it might be better to do a `go build` instead. Will update now.

**Comment by walle** [comment]
I have updated the pull request now.

**Comment by DomT4** [comment]
Nobody screamed murder so merged in https://github.com/Homebrew/homebrew/commit/63260c4ef310b9390465b526b5d60a1ea7f40727. Thank you for your contribution to Homebrew @walle; we appreciate it! :smiley_cat:

**Comment by walle** [comment]
Thanks, that is great!

Have a nice day!

**Pull request closed without merging**